### PR TITLE
Review tiles

### PR DESCRIPTION
**Pull request by PBandJaimie**
@a-nouraldean @EJLEE808 @Taniel-P 

I've made a lot of progress on the display of the review tiles, still quite a bit of work to go, however.

### CODE REVIEW

File: client/src/components/Reviews/Images.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class Images extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
```
Comment (Taniel-P) [inline client/src/components/Reviews/Images.jsx:7]: The state is empty.  Should be src based on line 14?

File: client/src/components/Reviews/SingleTile.jsx [hunk 2]
```diff
@@ -1,20 +1,106 @@
...
-//recommend (props.review.recommend)
-//reviewer name (props.review.reviewer_name)
-//response to review props.review.response
-//rating helpfulness props.review.helpfulness
-//     return (
-//       <div>
-//         null
-//       </div>
-//     )
+class SingleTile extends React.Component {
```
Comment (Taniel-P) [inline client/src/components/Reviews/SingleTile.jsx:4]: This is great!

**Review by Taniel-P — CHANGES_REQUESTED** [review]
This is great work!  There are so many edge cases for you widget it's crazy, but everything looks awesome! 

It is saying there is a merge conflict though, can you fix that before we merge with main.

**Comment by PBandJaimie** [comment]
Just added scroll and button functionality.

**Review by Taniel-P — APPROVED** [review]
Changes look good!

**Pull request merged**